test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/Tests/PythonEventhubAppTest.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.Common;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.Tests;
+
+public class PythonEventhubAppTest : BaseE2E, IClassFixture<KafkaE2EFixture>
+{
+	private readonly KafkaE2EFixture _kafkaE2EFixture;
+	private readonly ITestOutputHelper _output;
+
+	public PythonEventhubAppTest(KafkaE2EFixture kafkaE2EFixture, ITestOutputHelper output) : base(kafkaE2EFixture,
```

Comment:
Btw, this is also a console logger, do you still need the logger in test case ? or one of them is not required ?

Reply:
Removed.